extendeddatacrossword_test.go

```diff
@@ -1,15 +1,22 @@
 package rsmt2d
 
 import (
-	"reflect"
+	"bytes"
+	"crypto/md5"
+	"math/rand"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestRepairExtendedDataSquare(t *testing.T) {
-	for codec := range CodecsMaxChunksMap {
+	for _, codec := range codecs {
+		codec := codec.codecType()
+
+		bufferSize := 64
 		original, err := ComputeExtendedDataSquare([][]byte{
-			{1}, {2},
-			{3}, {4},
+			makeCheckedRandBytes(bufferSize), makeCheckedRandBytes(bufferSize),
+			makeCheckedRandBytes(bufferSize), makeCheckedRandBytes(bufferSize),
```

Comment:
Maybe we should change this to:
```go
		repeatedOnes := bytes.Repeat([]byte{1}, bufferSize)
		repeatedTwos := bytes.Repeat([]byte{2}, bufferSize)
		repeatedThrees := bytes.Repeat([]byte{3}, bufferSize)
		repeatedFours := bytes.Repeat([]byte{4}, bufferSize)

		original, err := ComputeExtendedDataSquare([][]byte{
			repeatedOnes, repeatedTwos,
			repeatedThrees, repeatedFours,
		}, codec)
``` 
to have deterministic tests here (and do a test with more realistic e.g. random data somewhere else.